testutils/check-calling-form-in-baseline/main.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
Could you also add instructions on how to run this in the terminal so people can run it before they send out a PR? (And we could then potentially also make it part of the gradle test rule) Assume people have Go installed, but may need to know how to set `$GOPATH`

Reply:
In README